pootle/apps/import_export/management/commands/export.py

```diff
@@ -17,19 +17,35 @@
 from pootle_project.models import Project
 from pootle_store.models import Store
 
+from ...utils import TPTMXExporter
+
 
 class Command(PootleCommand):
     help = "Export a Project, Translation Project, or path. " \
            "Multiple files will be zipped."
 
     def add_arguments(self, parser):
         super(Command, self).add_arguments(parser)
-        parser.add_argument(
+        group = parser.add_mutually_exclusive_group()
```

Comment:
Not critical but naming the group according to its purpose would help code readability `group_path_or_tmx`